src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Shared/DraftApprenticeship.cshtml

```diff
@@ -533,7 +533,12 @@
         return indexed_array;
     }
 
-    document.addEventListener("DOMContentLoaded", showTrainingDetailsIfRequired);
+    function onFormSubmitted(e) {
+        e.preventDefault();
+        if ($("#draftApprenticeshipSection1").hasClass("das-hide") === false)
+            continueJourney();
+        else return;
```

Comment:
Does returning in this function end up calling the default form submit behaviour?